org.eclipse.dawnsci.nexus.test/src/org/eclipse/dawnsci/nexus/ComplexNexusFileTest.java

```diff
@@ -54,13 +55,13 @@ private void setupTestData() {
 		testData.put("actualTime", createDoubleDataset(
 				1.39112723398E9, 1.391127234478E9, 1.391127234673E9, 1.391127234871E9, 1.391127235068E9,
 				1.391127248978E9, 1.391127249481E9, 1.391127249751E9, 1.391127249953E9, 1.391127250154E9));
-		testData.put("beamok", DatasetFactory.ones(shape, Dataset.FLOAT64));
+		testData.put("beamok", DatasetFactory.ones(shape));
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "beamok" 7 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)